### PR TITLE
Update contribution guide with info about forks (2.0-preview)

### PR DESCRIPTION
**Pull request by erinha**
Adding info to the CONTRIBUTING.md file in `2.0-preview` branch for how to fork the repository to contribute to the teams-js repo.

### CODE REVIEW

**Review by aeunms — APPROVED** [review]
LGTM.

**Review by Jessica-Koch — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,11 +5,13 @@ One of the best ways to contribute is to participate in discussions and discuss
...
+4. Commit your changes.
 5. Run `yarn changefile`, answer the following prompts, and push your changes to GitHub. **Important! Our pipeline checks will fail if you skip this step.** More on this below in our Changelog section.
-6. Create a pull request against the origin's `2.0-preview` branch. If you don't know what a pull request is read [this article](https://help.github.com/articles/using-pull-requests).
+6. Create a [pull request against the upstream's `2.0-preview` branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork). If you don't know what a pull request is read [this article](https://help.github.com/articles/using-pull-requests).
+7. A user with write access to the upstream repo will need to trigger the checks that run on pull requests in the teams-js repo.
+8. Once your pull request is approved, a user with write access will need to merge your pull request into the upstream repo. You can then delete your branch.
```
Comment (AE-MS) [inline CONTRIBUTING.md:14]: >You can then delete your branch. [](http://example.com/codeflow?start=125&length=32)

If you are going to have this recommendation, should it say "You can then delete your branch and/or fork?"

**Review by AE-MS — APPROVED** [review]
:shipit:

File: CONTRIBUTING.md (hunk 1 above)
Comment (erinha) [reply CONTRIBUTING.md:14]: I considered that, but you can continue making further changes on the same fork and those who plan to make changes on an ongoing frequency probably wouldn't be deleting the fork. But they can just ignore that part. :)

Comment (AE-MS) [reply CONTRIBUTING.md]: Yes, that's a valid point and supports my suggested use of "and/or"

**Review by AE-MS — APPROVED** [review]
:shipit:

**Pull request merged**